health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/InboxFilter.js

```
@@ -0,0 +1,80 @@
+      ? value.map((item) => ({
+          code: item?.[1]?.code,
+          name: item?.[1]?.name,
+        }))
+      : [];
+    setFilterValues((prev) => ({
+      ...prev,
+      [key]: transformedValue,
+    }));
+  };
+
+  let options = [
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'options' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)